### PR TITLE
TF-1142 Fix bug mark draft mail as read, and remove read/unread button from draft mail

### PR DESCRIPTION
**Pull request by sherlockvn**
#1142 

https://user-images.githubusercontent.com/43041967/200782588-31137bf4-b9d2-4024-969d-6211c92d7997.mov

### CODE REVIEW

**Comment by chibenwa** [comment]
While we are at it, do we still need the mark all as read button for the Draft mailbox?

**Comment by hoangdat** [comment]
@sherlockvn please remove this inside draft also

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart [hunk 1]
```diff
@@ -346,7 +348,9 @@ class MailboxDashBoardView extends BaseMailboxDashBoardView {
           ..onPressActionClick(() => controller.dispatchAction(SelectionAllEmailAction()))
           ..text(AppLocalizations.of(context).select_all, isVertical: false))
         .build(),
-      if (!searchController.isSearchEmailRunning)
+      if (!searchController.isSearchEmailRunning && 
+          mailBoxDashboardController.selectedMailbox.value != null && 
+          !mailBoxDashboardController.selectedMailbox.value!.isDrafts)
```
Comment (hoangdat) [inline lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart:353]: put it in separate method, it makes the code readable

**Review by dab246 — APPROVED** [review]
lgtm

**Review by hoangdat — APPROVED** [review]

**Pull request merged**